IGN.js

```diff
@@ -1,103 +1,105 @@
 {
-	"translatorID": "d210c5a1-73e1-41ad-a3c9-331d5a3ead48",
+	"translatorID": "cecafe6f-7fc9-4bdb-ae70-bd0a75a8ca52",
```

Comment:
Even if it completely replaces the former translator, without re-using any of it?